openassessment/xblock/test/test_submission.py

```diff
@@ -566,6 +569,47 @@ def test_get_download_urls_from_submission_single_key(self, xblock):
 
             mock_download_url.assert_has_calls([call('key-1')])
 
+    @scenario('data/submission_open.xml', user_id="Red Five")
+    def test_get_team_context_exceptions(self, xblock):
+        """
+        Unit tests for error behavior in get_team_submission_context
+        """
+        xblock.location = 'some-uuid'
+        self.setup_mock_team(xblock)
+
+        # If there's no teams config, just return without adding anyting to the context, but log an error
+        with LogCapture() as logger:
```

Comment:
My only comment is that this should maybe be 2 tests for clearer test output, but is a nit.